src/libifconfig.c

```diff
@@ -75,6 +76,7 @@ ifconfig_close(ifconfig_handle_t *h)
 			(void)close(h->sockets[i]);
 		}
 	}
+	freeifaddrs(h->ifap);
```

Comment:
Presumably this fixes a separate problem, so ideally it should be committed separately.